frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -397,12 +397,12 @@ export default memo(function HeatMap({
             />
             <CellTypeTagContainer>
               {filteredCellTypes.map((cellType) => (
-                <StyledTag
-                  label={cellType}
-                  key={cellType}
-                  onDelete={handleCellTypeDelete(cellType)}
-                  data-testid={`cell-type-tag-${cellType}`}
-                />
+                <span key={cellType} data-testid={`cell-type-tag-${cellType}`}>
```

Comment:
`data-testId` doesn't render on `StyledTag` - probably need to fix this in SDS by passing `...rest` onto `<Tag {...rest} />` 🙆‍♂️ 

CC: @masoudmanson